src/pages/zh-cn/core-concepts/framework-components.md

```diff
@@ -117,7 +121,7 @@ import MyVueComponent from '../components/MyVueComponent.vue';
 
 在 Astro 组件中，你可以向框架组件传递子组件。每个框架都有自己的模式来引用这些子组件：React、Preact 和 Solid 均使用一个特殊的属性名 `children`，而 Svelte 和 Vue 则使用 `<slot />` 元素。
 
-```astro
+```astro title="src/pages/named-slots.astro" /slot="(.*)"/
 ---
 // src/pages/MyAstroPage.astro
```

Comment:
And then this one is component children:

````suggestion
```astro {5}
---
// src/pages/component-children.astro
````